test/integration/verify/controls/azure_migrate_project_events.rb

```diff
@@ -0,0 +1,10 @@
+resource_group = input(:resource_group, value: '')
+project_name = input(:project_name, value: 'inspec-migrate-integ')
```

Comment:
You can pass this value from variable.tf.All the other resources using same convention.

Reply:
same as [here](https://github.com/inspec/inspec-azure/pull/450#discussion_r699333308)